src/core/CellComponent/index.tsx

```diff
@@ -14,7 +14,9 @@ const CellComponent = forwardRef(
     const { children } = props;
 
     return (
-      <StyledCellComponentData {...props}>{children}</StyledCellComponentData>
+      <StyledCellComponentData className="cell-component" {...props}>
```

Comment:
thanks for catching this

Reply:
When the TableRow is disabled, I needed this className to properly style the cell components!